### PR TITLE
Update documentation 

### PR DESCRIPTION
**Pull request by jryon**
Fix links, grammar, remove outdated info, and improve clarity.

### Description

This pull request is to address outdated info on readthedocs.

### CODE REVIEW

File: doc/source/calacs.rst [hunk 1]
```diff
@@ -55,37 +55,26 @@ ACSSUM Python Wrapper
...
-This functionality is provided but not run as part of the pipeline.
+This functionality is provided to simulate the CTE effects associated with
+readout of the WFC detectors. It is not part of the standard CALACS pipeline.
 
 .. automodapi:: acstools.acscteforwardmodel
     :no-heading:
     :headings: =-
     :no-inheritance-diagram:
 
-PIXVALUE in FITS File
```
Comment (pllim) [inline doc/source/calacs.rst:65]: Why is this removed?

Comment (jryon) [reply doc/source/calacs.rst:65]: Well, I'm not sure this section reflects our actual recommendations to users. I use astropy.io.fits with acstools all the time and have never run into an issue, and we have example analysis notebooks for ACS data that do the same. Also, CALACS will produce an extension when all pixel values are the same, at least in some cases: https://github.com/spacetelescope/hstcal/issues/371#issuecomment-1031986684

Perhaps I should rewrite the section to recommend stsci.tools.stpyfits if users run into an issue?

Comment (pllim) [reply doc/source/calacs.rst:65]: I think you will see this in RAW, where ERR and DQ are empty and represented by PIXVALUE. Or I think any calibrated data with homogeneous array, like if your DQ is somehow all zeroes.

Comment (jryon) [reply doc/source/calacs.rst]: True. I put this section back in and updated it to hopefully make it more clear what may happen when using astropy with pipeline products, and recommend rather than require stpyfits.

**Review by pllim — APPROVED** [review]
LGTM but you might want another ACS Team member to have a look too. Thanks!

File: doc/source/calacs.rst [hunk 2]
```diff
@@ -210,13 +213,11 @@ taken after January 2015.
 Pixel-Based CTE Correction (PCTECORR)
 -------------------------------------
 
-For all full-frame WFC exposures, pixel-based CTE correction (ACS ISR to be updated)
-is applied in ACSCTE that occurs after the ACSCCD series;
-i.e., after BLEVCORR.
+For all full-frame WFC exposures, pixel-based CTE correction (ACS ISR 2018-04)
```
Comment (rjavila) [inline doc/source/calacs.rst:216]: Should probably include link to ISR like in other instances in this documentation.

Comment (jryon) [reply doc/source/calacs.rst]: Cool, I've added a link here and to other ISRs cited in this file.

**Comment by jryon** [comment]
@pllim do you know why the link check is failing? Earlier I added other links that didn't fail the link check.

**Review by rjavila — APPROVED** [review]

File: doc/source/calacs.rst [hunk 3]
```diff
@@ -134,9 +135,8 @@ CALACS supports several command line options:
 * --ctegen <1|2>
 
     * Specify which generation CTE correction to use, the default is 2. Gen 1 (officially deprecated) refers to
-      the correction algorithm used in calacs version pre 9.2.0 in relation to the following ISRs
-      ACS ISR 2010-03 and 2012-03. Gen 2 refers to the new CTE correction algorithm implemented in calacs
-      version 9.2.0 (HSTCAL 1.3.0) in relation to the ISR (to be updated).
+      the correction algorithm used in calacs version pre 9.2.0 in relation to
+      `ACS ISR 2010-03 <https://ui.adsabs.harvard.edu/abs/2010PASP..122.1035A/abstract>`_. Gen 2 refers to the new CTE correction algorithm implemented in calacs version 9.2.0 (HSTCAL 1.3.0) in relation to `ACS ISR 2018-04 <https://ui.adsabs.harvard.edu/abs/2018acs..rept....4A/abstract`_.
```
Comment (pllim) [inline doc/source/calacs.rst:139]:
```suggestion
      `ACS ISR 2010-03 <https://ui.adsabs.harvard.edu/abs/2010PASP..122.1035A/abstract>`_. Gen 2 refers to the new CTE correction algorithm implemented in calacs version 9.2.0 (HSTCAL 1.3.0) in relation to `ACS ISR 2018-04 <https://ui.adsabs.harvard.edu/abs/2018acs..rept....4A/abstract>`_.
```
